src/views/mentorship/mentorship-hero/ui/mentorship-hero.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames/bind';
+import Image, { StaticImageData } from 'next/image';
+import { PAGE_NAMES } from '@/shared/constants';
+import { LinkCustom } from '@/shared/ui/link-custom';
+import { MainTitle } from '@/shared/ui/main-title';
+import { Subtitle } from '@/shared/ui/subtitle';
+import { heroPageData, mentorsRegisterData } from 'data';
+
+import styles from './mentorship-hero.module.scss';
+
+export const cx = classNames.bind(styles);
+
+export type MentorshipHeroProps = {
+  mainTitle: string;
+  widgetTitle: string;
+  subTitle: string[] | [];
+  heroImageSrc?: StaticImageData;
+  imageAltText: string;
+};
+
+type SubTitleProps = Pick<MentorshipHeroProps, 'subTitle'>;
+
+const HeroSubTitle = ({ subTitle }: SubTitleProps) => {
+  if (!subTitle?.length) {
+    return null;
+  }
+
+  return (
+    <div className={cx('subtitle-container')}>
+      {subTitle.map((item: string, index: number) => {
+        return (
+          <Subtitle key={index} fontSize="extra-small" weight="bold" color="black">
+            {item}
+          </Subtitle>
+        );
+      })}
+    </div>
+  );
+};
+
+export const MentorshipHero = () => {
+  const {
+    mainTitle,
+    subTitle = [],
+    heroImageSrc,
+    imageAltText = '',
```

Comment:
I'm not sure that we have to use this props

Reply:
Deleted